deploy_template/.dragonruby/stubs/html5/dragonruby-html5-loader.js

```diff
@@ -542,73 +542,26 @@ var Module = {
     startGame();  // go go go!
   },
   startClickToPlay: function() {
-    var base64 = base64Encode(FS.readFile(GDragonRubyIcon, {}));
-    var div = document.createElement('div');
-    var leftPx = ((window.innerWidth - 640) / 2);
-    var leftPerc = Math.floor((leftPx / window.innerWidth) * 100);
-    div.id = 'clicktoplaydiv';
-    div.style.width = '50%';
-    div.style.height = '50%';
-    div.style.backgroundColor = 'rgb(40, 44, 52)';
-    div.style.position = 'absolute';
-    div.style.top = '50%';
-    div.style.left = '50%';
-    div.style.transform = 'translate(-50%, -50%)';
-
-    var img = new Image();
-    img.onload = function() {  // once we know its size, scale it, keeping aspect ratio.
-      var pct = 30;
-      var w = img.naturalWidth;
-      var h = img.naturalHeight;
-      if (!w || !h || (w == h)) {
-        img.style.width = '' + pct + '%';
-        img.style.height = '' + pct + '%';
-      } else if (w > h) {
-        img.style.width = '' + pct + '%';
-      } else {
-        img.style.height = '' + pct + '%';
-      }
-      img.style.display = 'block';
-    }
```

Comment:
I could not sanely reproduce this Javascript that dynamically sets the icon CSS depending on its actual width/height.

But I replaced it with static styles (fixed `max-height` `max-height` of 30%) that should hopefully have the same effects.....

I can try and take some comparison screenshots later with some extremely proportioned icons